tools/sdk-readme-table-generator/TableGenerator/Program.fs

```diff
@@ -9,6 +9,9 @@ let inputBranches =
         [ { GitBranchName = "master"
             DisplayName = "Master<br>(6.0.x&nbsp;Runtime)"
             AkaMsChannel = Some("net6/dev") }
+          { GitBranchName = "release/6.0.1xx-preview1"
+            DisplayName = "Release/6.0.1xx-preview1<br>(6.0.x&nbsp;Runtime)"
+            AkaMsChannel = Some("net5/6.0.1xx-preview1/daily") }
```

Comment:
shouldn't this be under a `net6` folder?

Reply:
Yes, thank you!